torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -92,6 +92,7 @@ bool should_allow_numbers_as_tensors(const std::string& name) {
       "sub",          "sub_",          "sub_out",
       "subtract",     "subtract_",     "subtract_out", // alias of sub
       "true_divide",  "true_divide_",  "true_divide_out",
+      "to",           "_to_copy",      "copy_",
```

Comment:
What's this for? This seems unrelated to this PR

Reply:
This was to get certain tests passing